airbyte-workers/src/test/java/io/airbyte/workers/temporal/scheduling/testsyncworkflow/SleepingSyncWorkflow.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.scheduling.testsyncworkflow;
```

Comment:
this package name seems weird. I think we can drop `test` and just use `sync` to match the `SyncWorkflow` package.

Reply:
The goal is to have some dedicated test `SyncWorkflow` that will be used for testing the `scheduling` package and not the `SyncWorkflow`. I added `test` because those workflow will only exist in the test folder.